### ISSUE TITLE
Memory Leak / OOM causing frequent Crash Loop on vizier-pem, related to gzip http?

### ISSUE THREAD

**Issue by jmcdonagh** [pixie-io/pixie#394]
**Describe the bug**

This is happening on select pem pods. I think it has something to do with gzip encoded http. On the pods that keep crashing I have tons of this message over and over:

```
W0120 17:22:49.358582 1454939 stitcher.cc:74] Unable to gunzip HTTP body.
```

This is happening on different clusters too. I tried bumping the DS to have a 3Gi limit just in case, and it doesn't help. Still eats that up.

**To Reproduce**
1. Install pixie and let it fly

**Expected behavior**
Expect for vizier-pem to not run out of memory and crash.

**Screenshots**
If applicable, add screenshots to help explain your problem. Please make sure the screenshot does not contain any sensitive information such as API keys or access tokens.

**Logs**

I can send the log zip on Slack, my name is Joe McDonagh on there. Feel free to ping me.

**App information (please complete the following information):**
- Pixie version

⮕  px version
Pixie CLI
0.7.2+Distribution.91e72cd.20211217191238.1

- K8s cluster version

⮕  k version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.4", GitCommit:"c96aede7b5205121079932896c4ad89bb93260af", GitTreeState:"clean", BuildDate:"2020-06-17T11:41:22Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.7", GitCommit:"132a687512d7fb058d0f5890f07d4121b3f0a2e2", GitTreeState:"clean", BuildDate:"2021-05-12T12:32:49Z", GoVersion:"go1.15.12", Compiler:"gc", Platform:"linux/amd64"}

Also happening on a cluster with this server version:

Server Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.5", GitCommit:"aea7bbadd2fc0cd689de94a54e5b7b758869d691", GitTreeState:"clean", BuildDate:"2021-09-15T21:04:16Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"linux/amd64"}

- Node Kernel version

5.11.0-1019-aws

- Browser version

N/A

**Additional context**

**Comment by oazizi000** [comment]
The gzip messages are most likely unrelated, but there was a known issue with memory spikes that was fixed in cd6f1bdf. That may address this issue. Let's re-evaluate after that gets rolled out (which should be sometime today).

**Comment by oazizi000** [comment]
@jmcdonagh Fixes for this went out for this a while back, and I believe you reported on our community slack that the issue looked fixed. I will go ahead and close this issue.

Please re-open the issue if the issue is not resolved.
